tests/integration/framework/test_crawlers.py

```diff
@@ -9,3 +11,78 @@ def test_deploys_database(sql_backend, inventory_schema):
     deployer.deploy_schema()
     deployer.deploy_table("grants", Grant)
     deployer.deploy_view("grant_detail", "queries/views/grant_detail.sql")
+
+
+def test_runtime_backend_incorrect_schema_and_table_handled(ws, wsfs_wheel):
+    commands = CommandExecutor(ws.clusters, ws.command_execution, lambda: ws.config.cluster_id)
+
+    commands.install_notebook_library(f"/Workspace{wsfs_wheel}")
+    query_response_incorrect_schema = commands.run(
+        """
+        from databricks.labs.ucx.framework.crawlers import RuntimeBackend
+        from databricks.sdk.errors import NotFound
+        backend = RuntimeBackend()
+        try:
+            backend.execute("USE ABCDEF")
```

Comment:
Use make_random() fixture to generate a name

Reply:
Done! However could not do the same in unit tests as I got linting errors and make fmt removed the import statement.